src/test/java/data/provider/ActivityProvider.java

```diff
@@ -0,0 +1,27 @@
+package data.provider;
+
+import com.github.javafaker.Faker;
+import common.ValueChoosers;
+import data.models.activites.CreateActivityRequest;
+import data.models.activites.UpdateActivityRequest;
+
+public class ActivityProvider {
+
+    public static CreateActivityRequest createActivityRequest() {
+        CreateActivityRequest createActivityRequest = new CreateActivityRequest();
+        createActivityRequest.setId(ValueChoosers.getRandomId());
+        createActivityRequest.setTitle(Faker.instance().book().title());
+        createActivityRequest.setDueDate(ValueChoosers.getRandomDueDate());
+        createActivityRequest.setCompleted(Faker.instance().random().nextBoolean());
+        return createActivityRequest;
+    }
+
+    public static UpdateActivityRequest updateActivityRequest() {
```

Comment:
Deluje mi da je Update isti kao i Create, ajde to napravi da ide kroz jedan request. ActivityRequest koji ces koristiti i za create i za update, na taj nacin smanjujemo duplirani kod